src/shared/calendarspec.c

```diff
@@ -872,6 +913,38 @@ static int parse_calendar_time(const char **p, CalendarSpec *c) {
         return 0;
 }
 
+static int parse_and_every(const char *s, usec_t *ts, usec_t *rep) {
```

Comment:
Please prefix arguments for storing results with `ret_`.
Please add assertion for the arguments.